# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/openjdk/acceptable-bootjdk-versions.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- jdk-jdk-25-ga/make/conf/version-numbers.conf.vanilla	2025-09-18 14:20:35.650225945 +0200
+++ jdk-jdk-25-ga/make/conf/version-numbers.conf	2025-09-18 14:20:45.084933290 +0200
@@ -37,6 +37,6 @@
 DEFAULT_VERSION_CLASSFILE_MAJOR=69  # "`$EXPR $DEFAULT_VERSION_FEATURE + 44`"
 DEFAULT_VERSION_CLASSFILE_MINOR=0
 DEFAULT_VERSION_DOCS_API_SINCE=11
-DEFAULT_ACCEPTABLE_BOOT_VERSIONS="24 25"
+DEFAULT_ACCEPTABLE_BOOT_VERSIONS="23 24 25 26"
 DEFAULT_JDK_SOURCE_TARGET_VERSION=25
 DEFAULT_PROMOTED_VERSION_PRE=
